Scripts/installAllure.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Usage: ./installAllure.sh
+# This script will install Allure dependencies
+
+echo -e "👉 Install allurectl"
+DOWNLOAD_URL="https://github.com/allure-framework/allurectl/releases/download/1.22.1/allurectl_darwin_amd64"
+curl -sL "${DOWNLOAD_URL}" -o ./fastlane/allurectl
+chmod +x ./fastlane/allurectl
+
+echo -e "👉 Install xcresults"
+DOWNLOAD_URL="https://github.com/eroshenkoam/xcresults/releases/download/1.10.1/xcresults"
+curl -sL "${DOWNLOAD_URL}" -o ./fastlane/xcresults
+chmod +x ./fastlane/xcresults
```

Comment:
Where're we using this script? I can't see it mentioned anywhere

Reply:
Good catch 👍, it now lives in `bootstrap.sh`